module/VuFind/src/VuFind/Search/Summon/Results.php

```diff
@@ -323,7 +323,7 @@ public function getSpellingSuggestions()
      *
      * @return array|bool false if no recommendations, detailed array otherwise.
      */
-    public function getBestBets()
+    public function getBestBets(): array|bool
```

Comment:
I'm tempted to suggest changing all of these `array|bool` types to `?array`, but I have no way of testing Summon code at this point, so it's probably safer not to change more than we have to.